### PR TITLE
add cantata.pacscript

### PR DESCRIPTION
**Pull request by ty1118**
after help from devs, this script is finally ready

### CODE REVIEW

File: packages/cantata/cantata.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name="cantata"
+pkgname="cantata"
+version="2.4.2"
+url="https://github.com/CDrummond/cantata/releases/download/v$version/cantata-$version.tar.bz2"
+depends="mpd media-player-info fonts-font-awesome libcddb2 qttools5-dev qtbase5-dev qtchooser qtmultimedia5-dev qt5-qmake qtbase5-dev-tools  libqt5svg5 libebur128-1 libmusicbrainz5cc2v5 libqt5multimedia5"
+description="Cantata is a graphical client for MPD. It is support for Linux, MacOSX, and Windows, Multiple MPD collections and it has a Highly customizable layout."
+hash="3ed25d0c838720af7ed08b89a30f3e05d6952d40a5f520f29b3558b2eacdec82"
+
```
Comment (wizard-28) [inline packages/cantata/cantata.pacscript:8]: Needs a `maintainer` field. I'll explain about it in discord.

File: packages/cantata/cantata.pacscript [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+build() {
+        cmake -DCMAKE_BUILD_TYPE=Release ..
+        make -j$(nproc)
+}
+
+install() {
+    sudo make install DESTDIR=${STOWDIR}/${name}
+}
```
Comment (wizard-28) [inline packages/cantata/cantata.pacscript:21]: spacing is inconsistent. We'll pick one in discord.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/cantata/cantata.pacscript (hunk 2 above)
Comment (ty1118) [reply packages/cantata/cantata.pacscript:21]: ok

**Comment by wizard-28** [comment]
A new PR will be made

**Pull request closed without merging**